argusconfig/page_list_reports.php

```diff
@@ -10,7 +10,7 @@
 
 	// Getting reports
 	$tableFields = 'd.id, d.path, d.reception, d.exported, d.contactName, d.contactPhoneNumber, d.disease, d.period, d.periodStart, d.reportId';
-	$SQL="SELECT " . $tableFields . ", group_concat(v.key SEPARATOR '|') AS dataKeys, group_concat(v.value SEPARATOR '|') AS dataValues FROM ses_data d INNER JOIN ses_datavalues v ON d.id = v.FK_dataId GROUP BY d.id ORDER BY reception DESC LIMIT 1000;";
+	$SQL="SELECT " . $tableFields . ", group_concat(v.key SEPARATOR '|') AS dataKeys, group_concat(v.value SEPARATOR '|') AS dataValues FROM ses_data d INNER JOIN ses_datavalues v ON d.id = v.FK_dataId GROUP BY d.id ORDER BY d.id DESC LIMIT 1000;";
```

Comment:
Could you please find how to keep the order by reception instead of ordering by id. This ordering is important to export data as old reports needs to be exported before more recents.